src/js/rx/containers/History.jsx

```diff
@@ -146,10 +146,13 @@ class History extends React.Component {
       content = (
         <div>
           <p className="rx-tab-explainer">Your VA prescription refill history.</p>
-          <SortMenu
-              onChange={this.handleSort}
-              options={fields}
-              selected={currentSort.value}/>
+          <div className="show-for-small-only">
```

Comment:
We've generally been using `@media (max-width: $medium-screen)` within the stylesheets to style the mobile-only content. Should we be using `show-for-small-only` classes instead?